stratum/hal/lib/tdi/tdi_sde_wrapper.cc

```diff
@@ -0,0 +1,3817 @@
+// Copyright 2019-present Barefoot Networks, Inc.
+// Copyright 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "stratum/hal/lib/tdi/tdi_sde_wrapper.h"
+
+#include <memory>
+#include <set>
+#include <utility>
+
+// P4OVS_CHANGES: Are all the disabled sections Tofino-specific?
+// If so, #ifndef P4OVS_CHANGES -> #ifdef TOFINO_TARGET.
+
+#include "absl/strings/match.h"
+#include "absl/synchronization/mutex.h"
+#include "absl/synchronization/notification.h"
+#include "absl/time/time.h"
+
+#ifdef TOFINO_TARGET
```

Comment:
This file in its current form does not compile for Tofino. We will need some interim fixes so that we can start testing this code on Tofino since that is the most urgent target.
I have a patch with DPDK_TARGET and TOFINO_TARGET all over to get it to compile for both targets although it is not the most desirable solution at this time.